app/models/friendship.rb

```diff
@@ -0,0 +1,14 @@
+class Friendship < ApplicationRecord
+  # belongs_to :friendship_user, class_name: 'User'
+  # belongs_to :friendship_folk, class_name: 'User'
+  belongs_to :user
+  belongs_to :friends, class_name: 'User', foreign_key: 'friendship_id'
```

Comment:
Since my partner and I used **friends** as source in the User model, it won't be a good idea if we remove the foreign_key: **'friendship_id'**.
Thank you.